### ISSUE TITLE
Logical AND between expressions (and not "and") returns error

### ISSUE THREAD

**Issue by sashaagafonoff** [rewbs/sd-parseq#100]
I have a strength dip pulse and a nested conditional so that I can skip a pulse if it lands too close to a scene change. If I use uppercase "AND" instead of lowercase "and", it errors with the following message:
_
> Error parsing interpolation for strength at frame 10 (if ((f-info_match_last("STRLEVEL")>=-1) AND (f-info_match_last("STRLEVEL")<5)) 1 else if (f-info_match_last("scene")<3) 0.35 else S-pulse(p=60,a=0.35,pw=3) ): Error: Syntax error at line 1 col 41: 1 0.35 ... [See Javascript console for full error message].
_
Here's the logic for reference:
```
if ((f-info_match_last("STRLEVEL")>=-1) AND (f-info_match_last("STRLEVEL")<3))
  S
else 
  if (f-info_match_last("scene")<3) 
    0.35
  else
    S-pulse(p=60,a=0.35,pw=3)
```

**Comment by rewbs** [comment]
Hi, parseq's syntax is case sensitive, so I'd expect uppercase AND to throw an error. To help me understand, do you have a particular concern with using lowercase 'and'?  Or is it that you'd just find it more user friendly if both upper and lower case worked?

**Comment by sashaagafonoff** [comment]
yes, once i'd worked it out, it was fine - i was flagging it mainly for useability - probably should have added that
